configman/option.py

```diff
@@ -95,6 +96,7 @@ def __init__(
         self.not_for_definition = not_for_definition
         self.reference_value_from = reference_value_from
         self.secret = secret
+        self.change_flag = change_flag
```

Comment:
Perhaps I'm being archaic but shouldn't this be a privately named instance attribute?